ffi/diplomat/js/examples/tinywasm/build.sh

```diff
@@ -21,18 +21,22 @@ BASEDIR=$(dirname "$(realpath "$0")")
 
 # Build the WASM library
 # TODO: This likely doesn't work if $BASEDIR has spaces
-RUSTFLAGS="-Cpanic=abort -Copt-level=s -C link-arg=-zstack-size=${WASM_STACK_SIZE} -Clinker-plugin-lto -Ccodegen-units=1 -C linker=${BASEDIR}/ld.py -C linker-flavor=wasm-ld" cargo +${ICU4X_NIGHTLY_TOOLCHAIN} build \
+RUSTFLAGS="-Cpanic=abort -Copt-level=s -C link-arg=-zstack-size=${WASM_STACK_SIZE} -Clinker-plugin-lto -Ccodegen-units=1 -C linker=${BASEDIR}/ld.py -C linker-flavor=wasm-ld" \
+# Special linker flags shouldn't overwrite the top-level target directory
+CARGO_TARGET_DIR="target" \
```

Comment:
I'm not sure if this works; what's wrong with the previous solution?

Reply:
This works.

Reply:
Actually is this ok? The `\` points at a line with a comment.

Reply:
Ah yes, this line worked but you're right, the rust flags got chopped. Fixed